cli/src/git_util.rs

```diff
@@ -172,7 +172,45 @@ pub fn with_remote_git_callbacks<T>(
     f(callbacks)
 }
 
-pub fn print_git_import_stats(ui: &mut Ui, stats: &GitImportStats) -> Result<(), CommandError> {
+pub fn print_git_import_stats(
+    ui: &mut Ui,
+    stats: &GitImportStats,
+    show_ref_stats: bool,
+) -> Result<(), CommandError> {
+    use std::collections::BTreeMap;
+
+    if show_ref_stats && !stats.new_remote_refs.is_empty() {
+        let mut new_remote_branches: BTreeMap<_, Vec<_>> = BTreeMap::new();
+        let mut tags = Vec::new();
+        let mut local_branches = Vec::new();
+        for refname in &stats.new_remote_refs {
+            match refname {
+                RefName::RemoteBranch { branch, remote } => {
+                    new_remote_branches.entry(remote).or_default().push(branch);
+                }
+                RefName::Tag(tag) => tags.push(tag),
+                RefName::LocalBranch(b) => local_branches.push(b),
+            }
+        }
+        local_branches.sort_unstable();
```

Comment:
nit: no need to sort the list again? The original set is sorted by `RefName`.

FWIW, I personally don't mind if branches and tags are displayed in one big list, something like:
```
branch: main@origin (tracked)
branch: feature1@origin (deleted)
..
tag: 1.2.3 (created)
```

It will simplify the post processing code.

Reply:
Done